marketplace/cmd/marketplace.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var githubAccessToken = os.Getenv("GITHUB_ACCESS_TOKEN")
```

Comment:
the access token should be a optional parameter

Reply:
This is also a valid way to provide the access token to the app. but i found out that env variable `GITHUB_TOKEN` is being used in several places like github ci and [gh cli app](https://cli.github.com/manual/gh_help_environment) and therefore a more standardized approach.